src/main/java/org/jabref/logic/cleanup/MoveFilesCleanup.java

```diff
@@ -31,17 +31,21 @@
     private final BibDatabaseContext databaseContext;
     private final FileDirectoryPreferences fileDirectoryPreferences;
 
-    private final LayoutFormatterPreferences layoutPrefs;
+    // FIXME: remove unused variables later S.G.
```

Comment:
Please remove the commented-out code here.

Reply:
Fixed.